app/models/repository.rb

```diff
@@ -162,13 +170,13 @@ def latest_changeset
   def latest_changesets(path, _rev, limit = 10)
     if path.blank?
       changesets.find(:all, include: :user,
-                            order: "#{Changeset.table_name}.committed_on DESC, #{Changeset.table_name}.id DESC",
-                            limit: limit)
+                      order: "#{Changeset.table_name}.committed_on DESC, #{Changeset.table_name}.id DESC",
```

Comment:
Line is too long. [106/100]<br>Align the elements of a hash literal if they span more than one line.